api/error_handler.py

```
@@ -0,0 +1,350 @@
+                # Get local variables (be careful with sensitive data)
+                local_vars = {}
+                for var_name, var_value in frame.f_locals.items():
+                    try:
+                        # Only include simple types to avoid serialization issues
+                        if isinstance(var_value, (str, int, float, bool, type(None))):
+                            local_vars[var_name] = var_value
+                        elif isinstance(var_value, (list, dict, tuple)) and len(str(var_value)) < 200:
+                            local_vars[var_name] = str(var_value)[:200]
+                        else:
+                            local_vars[var_name] = f"<{type(var_value).__name__}>"
+                    except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)